4.0/es/0x01-Frontispiece.md

```diff
@@ -0,0 +1,46 @@
+# Frontispiece
+
+## Acerca del Estándar
+
+La norma de verificación de la seguridad de las aplicaciones (ASVS) es una lista de requisitos o pruebas de seguridad de las aplicaciones que pueden utilizar los arquitectos, desarrolladores, probadores, profesionales de la seguridad, proveedores de herramientas y consumidores para definir, construir, probar y verificar aplicaciones seguras.
+
+## Copyright y Licencia
+
+Version 4.x (Bleeding Edge version), 2021
```

Comment:
Please can you confirm you definitely based off the correct version as I can see this line is different to what is in the 4.0.2 version: https://github.com/OWASP/ASVS/blob/v4.0.2/4.0/en/0x01-Frontispiece.md

Reply:
Dear Josh ...
Sorry, we made a mistake.

We, the team, could not properly copy and paste from our 100% Spanish translated PDF file. And then rewrite them as markdown files. 
Just for a coincidence. Right now we are completing the execution of a CybersecTour in several Latin countries ... And we are very close to finishing the talks.
As soon as we release collaborators, we will ask them to do a new, more intense and personal QA.

Best Regards.